x-pack/plugins/fleet/public/components/google_cloud_shell_guide.tsx

```diff
@@ -53,7 +53,7 @@ export const GoogleCloudShellGuide = (props: { commandText: string }) => {
                   defaultMessage="Copy the following command and replace <PROJECT_ID> with your project ID."
```

Comment:
I suggest we also update the instruction so the user knows it can be editable before copying:

```suggestion
                  defaultMessage="Replace <PROJECT_ID> in the following command with your project ID and copy the command."
```

Reply:
sounds good !